### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by gjdonkers** [aaubry/YamlDotNet#612] [Q&A] [answered]
Hi @aaubry,
I really love working Yaml and your project. But today I've found a problem serializing and deserializing a UTC DateTime. I think I've got a fix for it on my local branch but when I want to push to make a PR I got a 403 Permission denied.

How can I contribute?

Regards

**Comment by aaubry** [comment, marked as answer]
Hi @gjdonkers,

The error is because you do not have permission to push changes directly to this repository. Only me, the owner, currently have such permission.

In order to open a pull request, you will need to create a fork of this repository to your account, then push to **that repository**. After doing that, you will be able to open a pull request between the branch on your fork to the main repository. I found [this guide that explains the process](https://gist.github.com/Chaser324/ce0505fbed06b947d962) in more detail, and there's also the [official documentation](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).

Thanks for your interest!

**Reply by gjdonkers** [reply]
I've created the PR now, hopefully all is correct

https://github.com/aaubry/YamlDotNet/pull/613

**Reply by gjdonkers** [reply]
It might be a good idea to add the details of forking and pushing to the CONTRIBUTIONS.md file, just for those like me who have no clue...
